baseplate/server/wsgi.py

```diff
@@ -15,18 +20,18 @@
 except ImportError:
     # LoggingLogAdapter is from gevent 1.1+
```

Comment:
should we just "enforce" that we be on a newer version of baseplate by not catching this anymore?

Reply:
yeah, I've got a branch coming with some version restriction updates